portal/src/texts/core/elevation.md

```diff
@@ -1,3 +1,17 @@
 ## Elevasjon
 
-Vi ønsker at brukerne lett skal se hvilket innhold som er på ett nivå og hvilket som er over- eller underordnet. Til det bruker vi blant annet skygger, som skaper inntrykk av elevasjon. Vi bruker ulike skygger for å skape et hierarki.
+Prinsippet elevasjon handler om å løfte det som er viktig og skape et intuitivt informasjonshierarki. Det kan blant annet gjøres ved å jobbe med nivåer og dybde.
+
+Bruk av skygge indikerer at elementer blir aktivert. Forskyvning ved scroll skaper også dybde og legger viktig innhold høyere opp og nærmere brukeren. Man kan også skape nivåer ved bruk av tydelige størrelsesforhold mellom elementer.
+
+### Nøkkelord
+
+-   Nivåer
+-   Løfte viktig innhold
+-   Tydelig hierarki
+
+### I bruk
+
+Benytt skygger og endring av størrelse for å kommunisere hvilke elementer på siden som har større viktighet. Bruk av størrelsesforhold med høy kontrast styrer fokus mot viktig innhold og bidrar til å skape naturlige brudd eller pauser i oppsettet.
```

Comment:
Forslag: Vi bruker skygger og elementer som endrer størrelse for å vise hvilke elementer på siden som er viktigst. Når vi bruker størrelse og høy kontrast sammen, styrer vi fokuset mot det viktigste innholdet og skaper naturlige pauser i oppsettet.

Reply:
Her var meningen i andre setning at det er kontrasten i størrelse som skaper fokus (STOR ↔︎ <sub>liten</sub>). Noen forslag som får frem det bedre?

Reply:
Hm.. her vil jo eksemplene hjelpe ved å vise hva som skjer. Hvis vi sier: "Vi bruker kontraster i størrelse til å fokusere på viktig innhold. Slik skaper vi naturlige pauser i oppsettet." Fungerer det?